packages/manager-components/README.md

```diff
@@ -0,0 +1,25 @@
+# ⚛️⚡ Manager Components Library
+
+## Features
+
+Hosts the components used in manager
+Keep it as simple as possible
+
+## Add new component
+
+1. Create a new components foler in `src/components`
+2. Add your `stories.tsx` file
+3. Add your e2e test
+
+## Import component to another project
+
+The component is not builded so you can import directly from the workspace like this :
+`import Card from '@ovhcloud/manager-components'`
```

Comment:
Generalise the README or give more context.
```suggestion
The component is not builded so you can import directly the component named `Card` from the workspace like this :
`import Card from '@ovhcloud/manager-components'`
```
OR
```suggestion
The component is not builded so you can import directly from the workspace like this :
`import CmpopnentName from '@ovhcloud/manager-components'`
```

Reply:
👍